extensions/sns/README.md

```diff
@@ -9,90 +9,40 @@ dfx sns [subcommand] [flag]
-| [`validate`](#_dfx_sns_validate)                   | Checks whether the sns config file is valid.                   |
-| [`deploy`](#_dfx_sns_deploy)                       | Deploys SNS canisters according to the local config.           |
-| [`prepare-canisters`](#_dfx_sns_prepare-canisters) | Prepares dapp canister(s) for SNS decentralization.            |
-| [`propose`](#_dfx_sns_propose)                     | Submits a CreateServiceNervousSystem NNS Proposal.             |
-| `help`                                             | Displays usage information message for a specified subcommand. |
+| Command                                                            | Description                                                                   |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------------------- |
+| [`init-config-file validate`](#_dfx_sns_init-config-file_validate) | Checks whether the sns config file is valid.                                  |
+| [`deploy-testflight`](#_dfx_sns_deploy-testflight)                 | Creates a test deployment of the SNS canisters according to the local config. |
+| [`prepare-canisters`](#_dfx_sns_prepare-canisters)                 | Prepares dapp canister(s) for SNS decentralization.                           |
+| [`propose`](#_dfx_sns_propose)                                     | Submits a CreateServiceNervousSystem NNS Proposal.                            |
+| `help`                                                             | Displays usage information message for a specified subcommand.                |
 
 To view usage information for a specific subcommand, specify the subcommand and the `--help` flag. For example, to see usage information for `dfx sns validate`, you can run the following command:
 
 ``` bash
 dfx sns validate --help
 ```
 
-
-## dfx sns create
-
-Use the `dfx sns create` command to create an SNS configuration file. The configuration file specifies important, legally and financially relevant details about dapp decentralization.  The file leaves blank parameters such as token name; you will need to fill these in.
-
-### Basic usage
-
-``` bash
-dfx sns create
-```
-
-### Examples
-
-You can use the `dfx sns create` command to create and view a configuration file:
-
-``` bash
-dfx sns create
-less sns.yml
-```
-
-## dfx sns validate
+## dfx sns init-config-file validate
 
 Use the `dfx sns validate` command to verify that an SNS configuration file is well formed.
 
 ### Basic usage
 
 ``` bash
-dfx sns validate
+dfx sns init-config-file validate
 ```
 
-### Examples
+## dfx sns deploy-testflight
 
-You can use the `dfx sns validate` command to verify that a configuration template is valid.  It is not; it needs details such as token name:
+Use the `dfx sns deploy-testflight` command to create a testflight deployment of the SNS canisters according to the local configuration file.
```

Comment:
> create a testflight deployment

What is a "testflight"? That should be explained here.

This is partially explained in the next paragraph, but that paragraph only said what a testflight is NOT. Whereas, a brief explanation of what it ACTUALLY IS would be called for.

Reply:
I added a link to the documentation, what do you think?

Reply:
update: also expanded a bit on what a testflight is